BookingsApi/BookingsApi.IntegrationTests/Helper/TestDataManager.cs

```diff
@@ -75,6 +75,68 @@ public async Task<JusticeUser> SeedJusticeUser(string userName, string firstName
 
             return justiceUser.Entity;
         }
+        
+        public async Task<List<JusticeUser>> SeedJusticeUserList(string userName, string firstName, string lastName, 
+            bool isTeamLead = false)
+        {
+            await using var db = new BookingsDbContext(_dbContextOptions);
+
+            var userList = new List<JusticeUser> { };
+            for (int i = 0; i < 10; i++)
+            {
+                var justiceUser = db.JusticeUsers.Add(new JusticeUser
+                {
+                    ContactEmail = userName + i,
+                    Username = userName + i,
+                    UserRoleId = isTeamLead ? (int)UserRoleId.VhTeamLead : (int)UserRoleId.Vho,
+                    CreatedBy = $"integration{i}.test@test.com",
+                    CreatedDate = DateTime.Now,
+                    FirstName = firstName + i,
+                    Lastname = lastName + i,
+                });
+                
+                userList.Add(justiceUser.Entity);
+                _seededJusticeUserIds.Add(justiceUser.Entity.Id);
+            }
+            await db.SaveChangesAsync();
+            return userList;
+        }
+        
+        public async Task<List<JusticeUser>> SeedAllocatedJusticeUserList(string userName, string firstName, string lastName)
+        {
+            await using var db = new BookingsDbContext(_dbContextOptions);
+
+            var userList = new List<JusticeUser> { };
+            for (int i = 0; i < 10; i++)
+            {
+                var justiceUser = db.JusticeUsers.Add(new JusticeUser
+                {
+                    ContactEmail = userName + i,
+                    Username = userName + i,
+                    UserRoleId = (int)UserRoleId.Vho,
+                    CreatedBy = $"integration{i}.test@test.com",
+                    CreatedDate = DateTime.Now,
+                    FirstName = firstName + i,
+                    Lastname = lastName + i,
+                });
+
+                if (i == 9)
```

Comment:
don't think we need this anymore do we. If you call this method I would expect it to create and allocate all of the users

Reply:
not really. I need a mix to test the filtering